pkgs/development/tools/protoc-gen-prost-crate/default.nix

```
@@ -0,0 +1,28 @@
+{ fetchCrate
+, lib
+, rustPlatform
+, protobuf
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "protoc-gen-prost-crate";
+  version = "0.3.1";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-MtGeU2PnVYPXb3nly2UaryjmjMz1lxcvYDjFiwf58FA=";
+  };
+
+  cargoSha256 = "sha256-dcKJRX/iHIWEmBD2nTMyQozxld8b7dhxxB85quPUysg=";
+
+  cargoBuildFlags = ["--bin" pname];
```

```suggestion
  cargoBuildFlags = [ "--bin" pname ];
```

Just to keep the style consistent within the file.